apps/ledger-live-desktop/src/renderer/screens/platform/LiveApp.tsx

```diff
@@ -27,10 +28,11 @@ type Props = {
     customDappUrl?: string;
   };
 };
-export default function PlatformApp({ match, appId: propsAppId, location }: Props) {
+
+export function LiveApp({ match, appId: propsAppId, location }: Props) {
   const history = useHistory();
   const { params: internalParams, search } = location;
-  const { state: urlParams, customDappUrl } = useLocation();
+  const { state: urlParams, customDappUrl } = useLocation<{ returnTo: string }>();
```

Comment:
- Where does `customDappUrl` come from?
- Is it Electron's deeplink, react-router-dom or both?
- What's the difference between `Prop["location"]` and return value of `useLocation`?